lib/registries/addon/drafts/controller.ts

```diff
@@ -0,0 +1,7 @@
+import { service } from '@ember-decorators/service';
+import Controller from '@ember/controller';
+import Analytics from 'ember-osf-web/services/analytics';
+
+export default class Drafts extends Controller {
```

Comment:
You should be able to delete all these controllers. A lot of pages don't require a controller at all, so we typically don't add them until they're needed.